src/OrchardCore/OrchardCore.Abstractions/Json/JOptions.cs

```diff
@@ -16,6 +16,7 @@ public static class JOptions
         PathStringJsonConverter.Instance,
         TimeSpanJsonConverter.Instance,
         DateTimeJsonConverter.Instance,
+        new JsonStringEnumConverter()
```

Comment:
First, you should create a static instance of this converter and the static instance. This will fix the issue. However, this will change the behavior of how we store enum moving forward from int to string. 

But it could increase the size of data being transmitted from and to the server. Personally, I am okay with storing strings instead of int since this will ensure we don't have enum problem if someone adds a new enum value in position other than the end of the enum.

I would also document this change in the release notes to ensure people are aware of this change.

@sebastienros thought on this global behavior?

Reply:
Related discussion https://github.com/OrchardCMS/OrchardCore/discussions/13154

Reply:
If we don't own the converter it's fine to not have the Instance property.
Does it work with both int and string as input, to be sure it's not breaking any.

Reply:
We don't own the converter

> Does it work with both int and string as input, to be sure it's not breaking any.

It should, but I will try another test

Reply:
@sebastienros yes it works with both int and string. However, it will write string moving forward and not int. It won't break anyone but will change the behavior.